docker/Dockerfile

```diff
@@ -94,6 +97,7 @@ RUN ln -s /ensembl-vep-release-${VEP_VERSION}/vep /vep
 WORKDIR /seqr-loading-pipelines
 COPY requirements.txt .
 RUN python3 -m pip install -r ./requirements.txt
+RUN python3 -m pip cache purge
```

Comment:
Can we just do `pip install  --no-cache-dir ` on the line above?

Reply:
That's just a matter of choice I guess (your choice as maintainer). Either `--no-cache-dir` needs to be added to every `pip install` command or one `pip cache purge` at the end.

Reply:
👍 looks like the only place we're missing it would be [here](https://github.com/broadinstitute/seqr-loading-pipelines/pull/632/files#diff-f34da55ca08f1a30591d8b0b3e885bcc678537b2a9a4aadea4f190806b374ddcR60), which may not even be necessary... 🤔 (I think `gustil` comes bundled with `gcloud` but I will double check).

Reply:
Yep, the `pip install of gsutil` appears unnecessary.

Reply:
I'm going to approve and make that change on a different branch.

Reply:
There are two more `pip install` commands wo `--no-cache-dir`. [here](https://github.com/broadinstitute/seqr-loading-pipelines/blob/ec6b322a104c6e086636c8619d0a8196915861ae/docker/Dockerfile#L99) and [here](https://github.com/broadinstitute/seqr-loading-pipelines/blob/ec6b322a104c6e086636c8619d0a8196915861ae/docker/Dockerfile#L51)

Reply:
I know I'm late on this one (sorry), but the `pip cache purge` here is ineffective unless it's chained onto the end of the pip installs with an `&&`: The cache contents introduced by the pip installs in the previous layers will continue to exist in those layers unless it's removed in the same `RUN` directive. Clearing the cache in its own step will make the final image appear cleaner when you run it, but all of the data in the previous layers still needs to be stored in the docker repository and shipped over the network when downloading the images.

`--no-cache-dir` on every pip call will actually accomplish reducing the final image size and saving bandwidth -- setting that option in a pip.conf file could be done if adding the flag feels too repetitive.

Alternatively, switching to a multi-stage build and not worrying about cache would be my preferred solution (see how we did this over in the broadinstitute/seqr Dockerfile).

Reply:
And, I see Ben has addressed this: https://github.com/broadinstitute/seqr-loading-pipelines/commit/5fb1be85f80cb244662fda056b4b8072b650611b 👍

Reply:
Thanks, I learned something.